ghost/members-events-service/test/event-storage.test.js

```diff
@@ -178,7 +181,7 @@ describe('EventStorage', function () {
             sinon.assert.calledTwice(subscribeSpy);
         });
 
-        it('filters if disabled', function () {
+        it('works with flag disabled', function () {
```

Comment:
I think I would just remove this test

Reply:
Ah cool will do, originally I did, but then decided to keep it since we are not bumping the flag to GA, just removing it for few cases, so this seemed like a nice check.

Reply:
Aha, that is true! Same for me, we can definitely keep it 👌